test/integration/discord.test.js

```diff
@@ -75,4 +78,75 @@ describe("test discord actions", function () {
         });
     });
   });
+
+  describe("test discord actions for nickname for verified user", function () {
+    beforeEach(async function () {
+      fetchStub = sinon.stub(global, "fetch");
+      superUser = { ...userData[4], discordId: "123456789" };
+      userId = await addUser(userData[0]);
+      superUserId = await addUser(superUser);
+      superUserAuthToken = authService.generateAuthToken({ userId: superUserId });
+    });
+
+    afterEach(async function () {
+      await cleanDb();
+      sinon.restore();
+    });
+
+    it("returns 200 for updating nickname patch method", function (done) {
+      fetchStub.returns(
+        Promise.resolve({
+          status: 200,
+          json: () => Promise.resolve({}),
+        })
+      );
+      chai
+        .request(app)
+        .patch(`/users/servername/${userId}`)
+        .send({ type: "discord" })
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body.message).to.equal("nickname has been changed");
+          return done();
+        });
+    });
+  });
+
+  describe("test discord actions for nickname for unverified user", function () {
```

Comment:
I suggest we don't need separate describe blocks for the same API. We can have a single describe block and have one it block for verified user expecting 200 response and another it block for unverified users expecting 403 response